mobile/test/java/io/envoyproxy/envoymobile/engine/EnvoyConfigurationTest.kt

```diff
@@ -160,13 +108,13 @@ class EnvoyConfigurationTest {
     assertThat(resolvedTemplate).contains("&h2_connection_keepalive_timeout 333s")
 
     // H3
-    assertThat(resolvedTemplate).contains(APCF_INSERT);
+    assertThat(resolvedTemplate).contains("http3_protocol_options:");
```

Comment:
I'm fine with this implementation, but we could also do `contains(JniLibrary.altProtocolCacheFilterInsert())` if we wanted to. I'm not sure if that's better (because it's checking for the full string and is resilient in the face of changes to that template string) or if it's worse (because it would hide bugs if we typo'd that template string).

Reply:
I'd prefer to leave as is because the end goal is to get rid of the template inserts entirely